static/nodejs_server/types.ts

```diff
@@ -32,6 +32,13 @@ export interface TileResult {
   title: string;
   // The type of the tile
   type: string;
+  // Dcids of the stat vars shown in the chart
+  svs: string[];
```

Comment:
from the standpoint of end-user readability of JSON, call it `vars` in place of `svs` maybe?

Reply:
done